Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim
+
+RUN apt-get -y update && apt-get -y install git wget build-essential python-setuptools python3-dev
+
+RUN mkdir /app
+
+RUN git clone https://github.com/activeloopai/Hub.git /app/ && \
+    cd /app/ && \
+    git checkout clean_2
```

Comment:
This should refer to a stable release branch or commit

Reply:
Good catch, will update to "release/2.0"